db/schema.rb

```diff
@@ -119,6 +119,28 @@
     t.index ["source_id"], name: "index_container_templates_on_source_id"
   end
 
+  create_table "containers", force: :cascade do |t|
+    t.bigint "tenant_id", null: false
+    t.bigint "container_group_id", null: false
+    t.string "name"
+    t.float "cpu_limit"
+    t.float "cpu_request"
+    t.bigint "memory_limit"
+    t.bigint "memory_request"
+    t.datetime "resource_timestamp"
+    t.jsonb "resource_timestamps", default: {}
+    t.datetime "resource_timestamps_max"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.datetime "source_created_at"
+    t.datetime "source_deleted_at"
+    t.datetime "archived_on"
+    t.index ["archived_on"], name: "index_containers_on_archived_on"
+    t.index ["container_group_id", "name"], name: "index_containers_on_container_group_id_and_name", unique: true
+    t.index ["container_group_id"], name: "index_containers_on_container_group_id"
```

Comment:
I think this index might be redundant, the unique index should be used for query ```.where(:container_group_id => XY)```

Reply:
:+1: